### PR TITLE
Adding Cassandra landing page

### LINKED ISSUE #80: Cassandra overview
Add initial product content following existing templates/conventions

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

We don't have a section on the devportal for Apache Cassandra yet, so this fixes #80 and gives us a place to add more Cassandra-related things under.

I took the liberty of declaring that "performant" is a word. My search engine results described it as "engineering jargon" which sounds perfect for this site ;)

### CODE REVIEW

File: docs/products/cassandra/index.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Aiven for Apache Cassandra
+==========================
+
+What is Apache Cassandra?
+-------------------------
+
+`Apache Cassandra <https://cassandra.apache.org/_/index.html>`_ is an open source, NoSQL database. It is specifically designed to be highly available, performant, and scalable. It supports a distributed architecture with asynchronous replication and is highly fault tolerant.
+
+Apache Cassandra originated originally from Facebook, and since then has had a strong history with some of the most demanding data platforms around while being in the care of the Apache project. From this strong history comes a robust database with an emphasis on the security expected, and the ability to scale up to meet the needs of even the largest data platforms.
```
Comment (TibsAtWork) [inline docs/products/cassandra/index.rst:9]: "originated originally" feels a little repetitive - perhaps "came originally"?

I'd maybe simplify the first sentence as a whole - perhaps something like:
```
Apache Cassandra came originally from Facebook. It has had a strong history with some of the most demanding data platforms around.
```

I'd maybe also change "the security expected" to just "security"

**Comment by laysauchoa** [comment]
hey, in the `Aiven developer` on [devportal](https://developer.aiven.io/) there is a section for Cassandra. Do you think it makes sense to link to the landing page like the others?

File path: `/devportal/index.rst`

**Comment by divdby0** [comment]
LGTM from product. Thanks.

File: docs/products/cassandra/index.rst (hunk 1 above)
Comment (TibsAtWork) [reply docs/products/cassandra/index.rst:9]: Or maybe "security expected" is meant to be "security aspect", which reads rather nicely.

**Review by TibsAtWork — APPROVED** [review]

**Pull request merged**